app/scripts/lib/auth-errors.js

```diff
@@ -116,6 +116,22 @@ define(function (require, exports, module) {
       errno: 127,
       message: t('Invalid authorization code')
     },
+    INVALID_PHONE_NUMBER: {
+      errno: 129,
+      message: t('Phone number invalid')
+    },
+    INVALID_PHONE_REGION: {
+      errno: 130,
+      message: t('Cannot send to this country')
+    },
+    SMS_ID_INVALID: {
+      errno: 131,
+      message: t('The Firefox Accounts developers screwed up')
```

Comment:
😮

Are we sure about this one? I know it's unlikely ever to occur but would this show up in the UI if it did?